call-home/src/common/constants.rs

```diff
@@ -1 +1,15 @@
+/// PRODUCT is the name of the project for which this call-home component is deployed.
 pub(crate) const PRODUCT: &str = "Mayastor";
+
+/// DATA_DIR_PATH is the directory path for the temporary files generated during
+/// encryption.
+pub(crate) const DATA_DIR_PATH: &str = "/encryption_dir";
```

Comment:
are these all at the root or are these relative paths?

Reply:
These are absolute paths. They are directories on the root directory.

Reply:
hmm how does that work for local testing, wouldn't it better to have a directory in the repo where those things live and reference it?